src/main/java/com/sap/sgs/phosphor/fosstars/data/github/HasSecurityPolicy.java

```diff
@@ -59,13 +66,15 @@ public HasSecurityPolicy update(ValueSet values) throws IOException {
       }
     }
 
-    values.update(new BooleanValue(HAS_SECURITY_POLICY, found));
-    return this;
+    Value<Boolean> value = HAS_SECURITY_POLICY.value(found);
+    cache.put(project, value);
```

Comment:
- Expiration clause for the cache
- cache() or cache

Reply:
Using the `cache` field is correct. Expiration time for cache entries is defined in the cache implementation (see my answer above).

Reply:
ok